waveshare2in13v2/example_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
```

Comment:
2021

Reply:
I left 2018 since the code largely comes from a file dated 2018, but I will change it.